include/structs/str_dungeon.h

```diff
@@ -12,6 +12,7 @@
 
 #define DUNGEON_MAX_SIZE_X 56
 #define DUNGEON_MAX_SIZE_Y 32
+#define DUNGEON_MAX_SIZE_X_MUL_Y (DUNGEON_MAX_SIZE_X * DUNGEON_MAX_SIZE_Y)
```

Comment:
X*Y is the max number of tiles in the dungeon. Perhaps a name like `DUNGEON_MAX_NUM_TILES` would be more clear than the literal `X_MUL_Y`.

Reply:
Applied your suggestion